tests/fullbench.c

```diff
@@ -268,6 +271,13 @@ static size_t benchMem(const void* src, size_t srcSize, U32 benchNb)
         if (g_zcc==NULL) g_zcc = ZSTD_createCCtx();
         break;
     case 12 :
+        if (g_zcc==NULL) g_zcc = ZSTD_createCCtx();
+        if (g_zcc) {
+            seqStore_t* seqStorePtr = ZSTD_getSeqStore(g_zcc);
+            if (seqStorePtr) seqStorePtr->forceExtDict = 1;
```

Comment:
Is it a problem? We have hidden functions so I think we can have hidden parameters. `seqStore_t` is not exposed in `zstd.h`. When we unlikely change `seqStore_t` we can change our testing programs.